src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,100 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
 import { formatNumber, Metric } from '../lib';
+import { MetricMeta } from '../lib/load_status';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
+  title: any;
+  description: any;
+}> = ({ testSubjectName, title, description }) => {
   return (
-    <EuiCard
-      data-test-subj={`serverMetric-${formatMetricId(metric)}`}
-      layout="horizontal"
-      title={formatMetric(metric)}
-      description={name}
+    <EuiStat
+      data-test-subj={testSubjectName}
+      title={title}
+      titleSize="xxs"
+      description={description}
+      reverse
     />
   );
 };
 
+/*
+ * Displays a metric with the correct format.
+ */
+export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
+  const { name } = metric;
+  if (name === 'Delay') {
+    return (
+      <EuiCard
+        data-test-subj={`serverMetric-${formatMetricId(metric)}`}
+        title={formatMetric(metric)}
+        textAlign="left"
+        description={`${name} avg`}
+        footer={
+          metric.meta && (
+            <MetricCardFooter
+              testSubjectName="serverMetricMeta"
+              title={formatDelayFooterTitle(metric.meta!)}
```

Comment:
I'm using the `title` to display custom metadata for the metrics that we want to show more detail for. Each metric has different details  so we need to format the `title` prop to pass in a string that we ultimately render.
I went with this option to make `MetricCardFooter` a little more generic, but being flexible enough to do some inline formatting within the main `MetricTile` component.